README.md

```diff
@@ -70,14 +72,14 @@ Example commands:
 - GitHub: [@githubhandle](https://github.com/adamilare)
 - Twitter: [@twitterhandle](https://twitter.com/mailtodare)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/adamilare)
-
+- GitUp Page: [Portfolio](https://adamilare.github.io/My-Portfolio/)
```

Comment:
- [ ] [OPTIONAL] It would be nice if you fix semantic mistake on text (`GitUp` => `GitHub`) and implement this link to another part like `Live Demo`

![image](https://user-images.githubusercontent.com/63915024/214055547-b849b502-fe51-41ea-9d0f-92e4beeab0b8.png)

Reply:
Thank you @mukhammadrizooff I will effect those changes in my next push.